Cargo.toml

```
@@ -14,7 +14,7 @@ bench = false
 
 [dependencies]
 smallvec = "1"
-robust = "0.2"
+robust = "1.1"
```

minor nit:
```suggestion
robust = "1.1.0"
```

see https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277

---

Thanks for the review! 😊